[Bug Report] [Build] [menu-item] 你们还能不能行了，每次发版全是bug 文档全是忽悠钱的广告，到底再搞什么啊，打包运行编译这么重大的报错都不测试吗

<!-- generated by https://issue.element-plus.org/ DO NOT REMOVE -->

Bug Type: **`Build`**

## Environment

- Vue Version: `3.2.35`
- Element Plus Version: `2.2.1`
- Browser / OS: ` Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.47`
- Build Tool: `Webpack`

## Reproduction

### Related Component

- `el-menu-item`

### Reproduction Link

[Github Repo](https://github.com/element-plus/element-plus)

### Steps to reproduce

 Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36 Edg/101.0.1210.47

### What is Expected?

./node_modules/_element-plus@2.2.1@element-plus/es/components/menu-item/style/css2.mjs 
Module not found: Error: Can't resolve 'element-plus/theme-chalk/menu-item.css' 

### What is actually happening?

./node_modules/_element-plus@2.2.1@element-plus/es/components/menu-item/style/css2.mjs 
Module not found: Error: Can't resolve 'element-plus/theme-chalk/menu-item.css' 

### Additional comments

你们还能不能行了，每次发版全是bug 文档全是忽悠钱挂羊头卖狗肉的广告，到底再搞什么啊
作为忠实的用户 真实没想到饿了么会沦落到如此地步 唉

./node_modules/_element-plus@2.2.1@element-plus/es/components/menu-item/style/css2.mjs 
Module not found: Error: Can't resolve 'element-plus/theme-chalk/menu-item.css' 

<!-- 如果你需要插入截图，请在这粘贴。 -->

--- a/packages/components/menu-item-group/style/css.ts
+++ b/packages/components/menu-item-group/style/css.ts
@@ -1,2 +1,2 @@
 import '@element-plus/components/base/style/css'
-import '@element-plus/theme-chalk/menu-item-group.css'
+import '@element-plus/theme-chalk/el-menu-item-group.css'

--- a/packages/components/menu-item/style/css.ts
+++ b/packages/components/menu-item/style/css.ts
@@ -1,2 +1,2 @@
 import '@element-plus/components/base/style/css'
-import '@element-plus/theme-chalk/menu-item.css'
+import '@element-plus/theme-chalk/el-menu-item.css'

